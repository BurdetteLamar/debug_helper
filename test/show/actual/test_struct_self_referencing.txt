---
DebugHelperTest::MyStruct (name='test_struct_self_referencing' size=3):
  Member 0:
    Name: :foo
    Value: 'DebugHelperTest::MyStruct #<struct DebugHelperTest::MyStruct foo=#<struct
      DebugHelperTest::MyStruct:...>, bar=1, baz=2>'
  Member 1:
    Name: :bar
    Value: Fixnum 1
  Member 2:
    Name: :baz
    Value: Fixnum 2
