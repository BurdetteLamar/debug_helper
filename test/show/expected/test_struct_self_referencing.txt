---
DebugHelperTest::MyStruct (message='test_struct_self_referencing'):
  DebugHelperTest::MyStruct#size: 3
  Member 0:
    Name:
      Symbol:
        Symbol#to_s: foo
        Symbol#size: 3
        Symbol#encoding: !ruby/encoding US-ASCII
    Value: 'DebugHelperTest::MyStruct #<struct DebugHelperTest::MyStruct foo=#<struct
      DebugHelperTest::MyStruct:...>, bar=1, baz=2>'
  Member 1:
    Name:
      Symbol:
        Symbol#to_s: bar
        Symbol#size: 3
        Symbol#encoding: !ruby/encoding US-ASCII
    Value: Fixnum 1
  Member 2:
    Name:
      Symbol:
        Symbol#to_s: baz
        Symbol#size: 3
        Symbol#encoding: !ruby/encoding US-ASCII
    Value: Fixnum 2
